x/authz/spec/01_concepts.md

```diff
@@ -0,0 +1,37 @@
+<!--
+order: 1
+-->
+
+# Concepts
+
+## Authorization
+Any concrete type of authorization defined in the `x/authz` module must fulfill `Authorization` interface outlined below. Authorizations determine exactly what privileges are granted. They are extensible and can be defined for any Msg service method even outside of the module where the Msg method is defined. Authorizations use the new `ServiceMsg` type from [ADR 031](docs/architecture/adr-31-msg-service.md).
```

Comment:
```suggestion
Any concrete type of authorization defined in the `x/authz` module must fulfill the `Authorization` interface outlined below. Authorizations determine exactly what privileges are granted. They are extensible and can be defined for any Msg service method even outside of the module where the Msg method is defined. Authorizations use the new `ServiceMsg` type from [ADR 031](../../../docs/architecture/adr-031-msg-service.md).
```

Reply:
I found relative path `../../../docs/architecture/adr-031-msg-service.md` showing 404 page on `https://docs.cosmos.network`.

Reply:
ah, ok. How did you test that, btw?

Reply:
I ran this command `npm run serve` from  docs folder.  Same issue exists [here](https://docs.cosmos.network/v0.40/docs/architecture/adr-003-dynamic-capability-store.html).